scheduler/src/main/resources/application.properties

```diff
@@ -11,16 +11,18 @@ scalingenginejob.reschedule.maxcount=6
 scalingengine.notification.reschedule.maxcount=3
 
 # scaling engine url
-autoscaler.scalingengine.url=http://test-scalingengine-test
+autoscaler.scalingengine.url=https://scalingengine
 
 #ssl
-server.ssl.key-store=../test-certs/scheduler.p12
+server.ssl.key-store=classpath:scheduler.p12
```

Comment:
I'm not sure what we were missing previously. Tomcat was picking the file only when classpath was specified. Still in application.properties could have left the relative path,. But TestConfiguration required the classpath for tomcat. After your comment tried giving the relative path just to be sure and it worked. Not sure what why was the behaviour different previously. Will change to relative path.